### ISSUE TITLE
Could not parse file

### ISSUE THREAD

**Issue by niahoo** [jfacorro/Eden#32]
Hi,

It seems that unicode characters are not parsed :
```edn
{
	:id "th"
	:name "Thaïlande"
	:geometry #wkt "POLYGON(1 2, 3 4, 5 6, 7 8)"
}
```

`=> FunctionClauseError`

Thank you

**Comment by jfacorro** [comment]
@niahoo Thank you for reporting this bug! I think I have discovered the problem and I will be publishing a new version of the library.

Thanks again for your help.

**Comment by niahoo** [comment]
Thank you !

**Comment by jfacorro** [comment]
There is now a new version of the library published to hex: [1.0.1](https://hex.pm/packages/eden/1.0.1). 

If you find any problems let me know.

**Comment by niahoo** [comment]
It works great :)
